src/Grpc.Net.Client/Balancer/Internal/ConnectionManager.cs

```diff
@@ -302,13 +302,7 @@ public void UpdateState(BalancerState state)
             }
         }
 
-        public async
-#if !NETSTANDARD2_0
-            ValueTask<(Subchannel Subchannel, BalancerAddress Address, Action<CompletionContext> OnComplete)>
-#else
-            Task<(Subchannel Subchannel, DnsEndPoint Address, Action<CompleteContext> OnComplete)>
-#endif
-            PickAsync(PickContext context, bool waitForReady, CancellationToken cancellationToken)
+        public async ValueTask<(Subchannel Subchannel, BalancerAddress Address, Action<CompletionContext> OnComplete)> PickAsync(PickContext context, bool waitForReady, CancellationToken cancellationToken)
```

Comment:
Why is it OK to remove the if-def here?

Reply:
Originally I wrote this code to support netstandard2.0. It ended up being net5.0 or later. This is an unused ifdef that didn't get removed.